mw-webapp/src/component/checkbox/Checkbox.tsx

```diff
@@ -0,0 +1,50 @@
+import styles from "src/component/checkbox/Checkbox.module.scss";
+
+/**
+ * Checkbox props
+ */
+interface CheckboxProps {
+  /**
+   * Checkbox status
+   */
+  checked: boolean;
+  /**
+   * Label's text
+   */
+  label: string;
+  /**
+   * Checkbox`s value
+   */
+  value?: string;
+  /**
+   * Checkbox's name
+   */
+  name?: string;
+  /**
+   * The flag sets the element to be required
+   */
+  required?: boolean;
+  /**
+   * Callback triggered on change checkbox
+   */
+  onChange: () => void;
```

Comment:
why does onChange signature deos not contain current checkbox state (isChecked?)

Reply:
rewrote component with encapsulate  state